assets/js/googlesitekit/widgets/components/Widget.js

```diff
@@ -22,21 +22,32 @@
 import classnames from 'classnames';
 import { node, string } from 'prop-types';
 
-const Widget = ( { children, slug } ) => {
+/**
+ * Internal dependencies
+ */
+import { WIDGET_WIDTHS } from '../datastore/constants';
+
+function Widget( { children, slug, width } ) {
 	return (
-		<div className={ classnames( 'googlesitekit-widget', `googlesitekit-widget--${ slug }` ) }>
+		<div className={ classnames( `googlesitekit-widget googlesitekit-widget--${ slug }`, {
+			'mdc-layout-grid__cell--span-12': width === WIDGET_WIDTHS.FULL,
+			'mdc-layout-grid__cell--span-6': width === WIDGET_WIDTHS.HALF,
+			'mdc-layout-grid__cell--span-3': width === WIDGET_WIDTHS.QUARTER,
+		} ) }>
 			{ children }
 		</div>
 	);
-};
+}
 
 Widget.defaultProps = {
 	children: undefined,
+	width: WIDGET_WIDTHS.QUARTER,
 };
 
 Widget.propTypes = {
 	children: node,
 	slug: string.isRequired,
+	width: string,
```

Comment:
These sizes should be handled as part of #1678, `width` shouldn't be passed to the `Widget` component.

Reply:
Ok, widget width changes have been reverted.